app/models/project/Page.scala

```diff
@@ -120,6 +123,44 @@ case class Page(override val id: Option[Int] = None,
 
 object Page {
 
+  private object ExternalLinkResolver {
+
+    class Factory(config: OreConfig) extends LinkResolverFactory {
+      override def getAfterDependents() = null
+      override def getBeforeDependents() = null
+      override def affectsGlobalScope() = false
+      override def create(context: NodeRendererContext) = new ExternalLinkResolver(this.config)
+    }
+
+  }
+
+  private class ExternalLinkResolver(config: OreConfig) extends LinkResolver {
+    override def resolveLink(node: Node, context: NodeRendererContext, link: ResolvedLink) = {
+      if (link.getLinkType.equals(LinkType.IMAGE) || node.isInstanceOf[MailLink]) {
+        link
+      } else {
+        link.withStatus(LinkStatus.VALID).withUrl(wrapExternal(link.getUrl))
+      }
+    }
+
+    private def wrapExternal(urlString: String) = {
+      try {
+        val uri = new URI(urlString)
+        if (uri.getScheme == null || "http" == uri.getScheme || "https" == uri.getScheme) {
```

Comment:
So you're saying that all links should be wrapped? even things like `mailto:` and `tel:`? with mailto it would allow all `@spongepowered.org` unless mail links were to be handled differently.